install-hooks.sh

```
@@ -0,0 +1,458 @@
+
+configure_workspace_trust() {
+    write_status "Configuring workspace trust..." "INFO"
+
+    local current_dir=$(pwd)
+
+    if [[ ! -f "$CLAUDE_STATE" ]]; then
+        write_status "No .claude.json found - trust will be prompted on first run" "WARN"
+        return
+    fi
+
+    # Backup existing state
```

Shellcheck (0.11.0)

[error] Literal carriage return. Run script through tr -d '\r' .

(SC1017)